scipy/misc/doccer.py

```diff
@@ -17,13 +16,4 @@ def __dir__():
 
 
 def __getattr__(name):
-    if name not in __all__:
-        raise AttributeError(
-            "scipy.misc.doccer is deprecated and has no attribute "
-            f"{name}.")
-
-    warnings.warn("The `scipy.misc.doccer` namespace is deprecated and "
-                  "will be removed in SciPy v2.0.0.",
-                  category=DeprecationWarning, stacklevel=2)
-
-    return getattr(_doccer, name)
```

Comment:
This looks wrong, you are removing a deprecation warning. It should still be emitted. Instead, update `__all__` in this file to match the contents of `scipy._lib.doccer`.